in-app-payments/src/main/java/eu/kevin/inapppayments/paymentconfirmation/PaymentConfirmationFragment.kt

```diff
@@ -44,6 +51,19 @@ internal class PaymentConfirmationFragment : BaseFragment<PaymentConfirmationSta
         return true
     }
 
+    private fun getKevinWebFrameColorsConfigurationFromTheme() : KevinWebFrameColorsConfiguration {
```

Comment:
I just wonder maybe it's worth to move theme config elsewhere as it is reused in both places and basically will be reused across all webviews. Together with `appendQueryParametersToUrl`. @aurimas-zarskis  do you think?

Reply:
Agree

Reply:
Also it's fine to pass that configuration from fragment, how it is now. Just the logic that fetches those colors should be moved to common class

Reply:
So redid the logic a bit, moved it out to a common place, but also got rid of appendQueryParametersToUrl in view models as Edgar suggested earlier.

Now the query parameters are passed as string to view model, tell me what you think about this.